lib/reek/documentation_link.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Reek
+  # Generate versioned links to our documentation
+  module DocumentationLink
+    HELP_LINK_TEMPLATE = 'https://github.com/troessner/reek/blob/v%<version>s/docs/%<item>s.md'.freeze
+
+    module_function
+
+    def build(item)
```

Comment:
Let's add some method documentation, just from looking at it i have no idea what "item" is.